src/scripts/wildBattle/PokemonsPerRoute.ts

```diff
@@ -26,7 +26,7 @@ const pokemonsPerRoute = {
             headbutt: [],
         },
         5: {
-            land: ['Pidgey', 'Pidgeotto', 'Rattata', 'Jigglypuff', 'Oddish', 'Meowth', 'Mankey', 'Abra', 'Bellsprout'],
+            land: ['Pidgey', 'Pidgeotto', 'Rattata', 'Jigglypuff', 'Oddish', '!(((owth', 'Mankey', 'Abra', 'Bellsprout'],
```

Comment:
?

Reply:
Probably hit the wrong stuff when editing the parenthesis as I was browsing throught the repo, will fix.

Reply:
Good :)

Reply:
It's fixed